### DISCUSSION TITLE
Downscaling volumes to consume less memory

### DISCUSSION THREAD

**Discussion by JumpLK** [Project-MONAI/MONAI#1963] [Q&A] [answered]
Hi,

I have some volumes that are too big to be processed without being downscale so I would like to know what are the different ways to do it with Monai.

**Comment by Nic-Ma** [comment, marked as answer]
Hi @JumpLK ,

I think maybe you can take a look at the PersistentDataset:
https://github.com/Project-MONAI/MONAI/blob/9f4da6acded249bba24c85eaee4ece256ed45815/monai/data/dataset.py#L95

Thanks.

**Reply by JumpLK** [reply]
Thank you :)
